eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -0,0 +1,54 @@
+# intended to be used as part of a release process
+parameters:
+  ArtifactLocation: 'not-specified'
+  PackageRepository: 'not-specified'
+  ReleaseSha: 'not-specified'
+  RepoId: 'not-specified'
+  WorkingDirectory: ''
+  ScriptDirectory: eng/common/scripts
+  TargetDocRepoName: ''
+  TargetDocRepoOwner: ''
+  PRBranchName: 'smoke-test-rdme'
+  ArtifactName: ''
+  Language: ''
+  DocRepoDestinationPath: '' #usually docs-ref-services/
+
+steps:
+- pwsh: |
+    git clone https://github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
+    
+    try {
+      Push-Location ${{ parameters.WorkingDirectory }}/repo
+    
+      Write-Host "git checkout smoke-test"
+      git checkout smoke-test
+    } finally {
+      Pop-Location
+    }
+  displayName: Clone Documentation Repository
+  ignoreLASTEXITCODE: false
+
+- task: PowerShell@2
+  displayName: 'Apply Documentation Updates From Artifact'
+  inputs:
+    targetType: filePath
+    filePath: ${{ parameters.ScriptDirectory }}/update-docs-metadata.ps1
+    arguments: -ArtifactLocation ${{parameters.ArtifactLocation}} -Repository ${{parameters.PackageRepository}} -ReleaseSHA ${{parameters.ReleaseSha}} -RepoId ${{parameters.RepoId}} -WorkDirectory '${{parameters.WorkingDirectory}}' -DocRepoLocation "${{parameters.WorkingDirectory}}/repo" -Language "${{parameters.Language}}" -DocRepoContentLocation ${{ parameters.DocRepoDestinationPath }}
+    pwsh: true
+  timeoutInMinutes: 5
```

Comment:
Why explicitly set a 5 min timeout?

Reply:
Good point. No guarantee some of these clones will run this fast anyway.